### PR TITLE
refactor(common): move inventory to separate folder

### PR DESCRIPTION
**Pull request by sandersms**
Moving inventory from common to separate folder to allow for future additions such as capabilities protobuf and to support the autogeneration of protobuf files using 'buf'

### CODE REVIEW

File: mocks/InvClient.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ package mocks
 import (
 	context "context"
 
-	_go "github.com/opiproject/opi-api/common/v1/gen/go"
```
Comment (glimchb) [inline mocks/InvClient.go:11]: this doesn't look auto-generated, can you please run make ?

File: Makefile [hunk 2]
```diff
@@ -61,7 +61,7 @@ mock-generate:
 
 	# Generate mocks for imported protobuf clients too
 	mockery --config=mocks/.mockery.yaml --name=IPsecClient --srcpkg=github.com/opiproject/opi-api/security/v1/gen/go
-	mockery --config=mocks/.mockery.yaml --name=InventorySvcClient --srcpkg=github.com/opiproject/opi-api/common/v1/gen/go
```
Comment (glimchb) [inline Makefile:64]: are we also missing new line here for `mocks/InvClient.go` ? wdyt ?

**Review by glimchb — APPROVED** [review]
lgtm

File: Makefile (hunk 2 above)
Comment (sandersms) [reply Makefile:64]: Its higher up in the Makefile
mock-generate:
	@echo "  >  Starting mock code generation..."
	# Generate mocks for exported interfaces
	mockery --config=mocks/.mockery.yaml --name=Connector --dir=grpc
	mockery --config=mocks/.mockery.yaml --name=InvClient --dir=inventory

**Pull request merged**